### ISSUE TITLE
There should be a well-defined and serached by default location for the standard library and user provided libraries.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [agda/agda#576]
```
I'd suggest something along the lines of

http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html

so that there are both system-wide and user paths that are included by default.
```

Original issue reported on code.google.com by `tom.pri...@ualberta.net` on 22 Feb 2012 at 5:25

**Comment by nad** [comment]
Can we close this issue now?

**Comment by UlfNorell** [comment]
Yes, this is fixed by the library system

**Comment by asr** [comment]
I added the 2.4.4 milestone, so we can track when an enhancement/bug was made/fixed.

**Comment by andreasabel** [comment]
Yes, considered fixed, but observe #1785 and #1693.

**Comment by asr** [comment]
#1785 and #1693 were fixed.
